python/ground/client.py

```diff
@@ -114,193 +130,284 @@ def createEdgeVersion(self,
         if from_node_version_end_id > 0:
             body["fromNodeVersionEndId"] = from_node_version_end_id
 
-        return self._makePostRequest(endpoint, body)
+        response = self._make_post_request(endpoint, body)
+        if response is None:
+            return None
+        else:
+            return model.core.edge_version.EdgeVersion(response)
 
-    def getEdge(self, source_key):
-        return self._getItem("edges", source_key)
+    def get_edge(self, source_key):
+        response = self._get_item("edges", source_key)
+        if response is not None:
+            return model.core.edge.Edge(response)
 
-    def getEdgeLatestVersions(self, source_key):
-        return self._getItemLatestVersions("edges", source_key)
+    def get_edge_latest_versions(self, source_key):
+        return self._get_item_latest_versions("edges", source_key)
 
-    def getEdgeHistory(self, source_key):
-        return self._getItemHistory("edges", source_key)
+    def get_edge_history(self, source_key):
+        return self._get_item_history("edges", source_key)
 
-    def getEdgeVersion(self, id):
-        return self._getVersion("edges", id)
+    def get_edge_version(self, id):
+        response = self._get_version("edges", id)
+        if response is not None:
+            return model.core.edge_version.EdgeVersion(response)
 
     '''
     GRAPH METHODS
     '''
 
-    def createGraph(self, source_key, name, tags={}):
-        return self._createItem("graphs", source_key, name, tags);
-
-    def createGraphVersion(self,
-            graph_id,
-            edge_version_ids,
-            reference=None,
-            reference_parameters={},
-            tags={},
-            structure_version_id=-1,
-            parent_ids=[]):
+    def create_graph(self, source_key, name, tags=None):
+        if tags is None:
```

Comment:
Same here.

Reply:
And it seems with all of the `create` methods.

Reply:
Same as above